doc/high-level.md

```diff
@@ -0,0 +1,83 @@
+| `(deye 1d0 '(3 3))`                             | `eye(3)`           | `eye(3)`                          | Create a 3x3 identity array of double-float element type.                            |
+| `(from-diag a`                                  | `diag(a)`          | `diag(a)`                         | Create a square matrix from the diagonal entries in `a` with zeroes everywhere else. |
+| `(rand '(3 4))`                                 | `rand(3,4)`        | `random.rand(3,4)`                | Create a random 3x4 array.                                                           |
+
+### Basic Operations
+
+| MAGICL     | MATLAB   | NumPy            | Description                 |
+|------------|----------|------------------|-----------------------------|
+| `(@ a b)`  | `a * b`  | `a @ b`          | Matrix multiplication       |
+| `(.+ a b)` | `a + b`  | `a + b`          | Element-wise add            |
+| `(.- a b)` | `a - b`  | `a - b`          | Element-wise subtract       |
+| `(.* a b)` | `a .* b` | `a * b`          | Element-wise multiply       |
+| `(./ a b)` | `a./b`   | `a/b`            | Element-wise divide         |
+| `(.^ a b)` | `a.^3`   | `a**3`           | Element-wise exponentiation |
+
+### Linear Algebra
+
+| MAGICL                                                  | MATLAB            | NumPy                                                          | Description                                |
+|---------------------------------------------------------|-------------------|----------------------------------------------------------------|--------------------------------------------|
+| `(det a)`                                               | `det(a)`          | `linalg.det(a)`                                                | Determinant of matrix                      |
+| `(trace a)`                                             | `trace(a)`        | `trace(a)`                                                     | Trace (sum of diagonal elements) of matrix |
+| `(upper-triangular a)`                                  | `triu(a)`         | `triu(a)`                                                      | Upper triangular part of matrix            |
+| `(lower-triangular a)`                                  | `tril(a)`         | `tril(a)`                                                      | Lower triangular part of matrix            |
+| `(transpose a)`                                         | `a.'`             | `a.transpose()` or `a.T`                                       | Transpose of matrix                        |
+| `(conjugate-transpose a)` or `(dagger a)`               | `a'`              | `a.conj().transpose()` or `a.conj().T`                         | Conjugate transpose of matrix              |
+| `(inv a)`                                               | `inv(a)`          | `linalg.inv(a)`                                                | Inverse of matrix                          |
```

Comment:
This is undefined at the moment (I'm ok with that, as long as it is marked as such).

Reply:
Ouch. I guess that is currently called `inverse`. Will switch to `inv`.

Reply:
524a087